e2e-tests/insurance/cypress/e2e/journeys/your-business/change-your-answers/change-your-answers-change-credit-control.spec.js

```diff
@@ -0,0 +1,73 @@
+import { summaryList } from '../../../../../../pages/shared';
+import { FIELD_VALUES } from '../../../../../../constants';
+import { INSURANCE_FIELD_IDS } from '../../../../../../constants/field-ids/insurance';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const { HAS_CREDIT_CONTROL } = INSURANCE_FIELD_IDS.EXPORTER_BUSINESS;
+
+const {
+  ROOT,
+  EXPORTER_BUSINESS: {
+    CREDIT_CONTROL_CHANGE,
+    CHECK_YOUR_ANSWERS,
+  },
+} = INSURANCE_ROUTES;
+
+const fieldId = HAS_CREDIT_CONTROL;
+
+context('Insurance - Your business - Change your answers - Credit control - As an exporter, I want to change my answers to the credit control section', () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      cy.startYourBusinessSection();
+
+      cy.completeAndSubmitCompanyDetails({});
+      cy.completeAndSubmitNatureOfYourBusiness();
+      cy.completeAndSubmitTurnoverForm();
+      cy.completeAndSubmitCreditControlForm({ hasCreditControlProcess: true });
+      cy.completeAndSubmitBrokerForm({ usingBroker: true });
+
+      url = `${Cypress.config('baseUrl')}${ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  describe('when clicking the `change` link', () => {
+    it(`should redirect to ${CREDIT_CONTROL_CHANGE}`, () => {
+      cy.navigateToUrl(url);
+
+      summaryList.field(fieldId).changeLink().click();
+
+      cy.assertChangeAnswersPageUrl(referenceNumber, CREDIT_CONTROL_CHANGE, HAS_CREDIT_CONTROL);
```

Comment:
will need to be updated after merging main-application-no-pdf

Reply:
Already started 😄